src/HasMedia/HasMediaCollections.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace Brackets\Media\HasMedia;
-
-use Spatie\MediaLibrary\HasMedia\HasMedia;
-
-interface HasMediaCollections extends HasMedia {
```

Comment:
@dejwCake the reason we are removing this interface is we don't use it anywhere or?